### ISSUE TITLE
typescript issues on yarn upgrade 

### ISSUE THREAD

**Issue by alee** [virtualcommons/port-of-mars#653]
Bumping versions on the server via `yarn upgrade --latest` causes compilation errors on the server. Not sure what changed, perhaps the typescript definitions for passport?

```
server_1  | Compilation error in /code/server/src/index.ts
server_1  | [ERROR] 06:16:26 ⨯ Unable to compile TypeScript:
server_1  | src/index.ts(69,24): error TS2769: No overload matches this call.
server_1  |   Overload 1 of 2, '(fn: (user: User, done: (err: any, id?: unknown) => void) => void): void', gave the following error.
server_1  |     Argument of type '(user: User, done: Function) => void' is not assignable to parameter of type '(user: User, done: (err: any, id?: unknown) => void) => void'.
server_1  |       Types of parameters 'user' and 'user' are incompatible.
server_1  |         Type 'User' is missing the following properties from type 'User': id, name, username, players, and 7 more.
server_1  |   Overload 2 of 2, '(fn: (req: IncomingMessage, user: User, done: (err: any, id?: unknown) => void) => void): void', gave the following error.
server_1  |     Argument of type '(user: User, done: Function) => void' is not assignable to parameter of type '(req: IncomingMessage, user: User, done: (err: any, id?: unknown) => void) => void'.
server_1  |       Types of parameters 'user' and 'req' are incompatible.
server_1  |         Type 'IncomingMessage' is missing the following properties from type 'User': id, name, username, players, and 7 more.
```

**Comment by cpritcha** [comment]
Fixed in 25248a56f349ad5a4492a8df2ea8d164fb0ff31c

**Comment by alee** [comment]
I think the upgrade broke some other things:

![image](https://user-images.githubusercontent.com/22534/104380985-14528e80-54e9-11eb-9ba7-69213f3d335f.png)

**Comment by alee** [comment]
this was caused by issues with `b-card` temporarily resolved in 6e651748f321016fcc3f57a6d85e6b782f40115d
